pkg/sql/logictest/testdata/logic_test/time

```diff
@@ -444,26 +444,29 @@ SELECT id FROM current_time_test WHERE
 > '1ms'::interval ORDER BY id ASC
 ----
 
-# test that current_time is correct in different timezones.
-
-statement ok
-set time zone +3
-
-statement ok
-create table current_time_tzset_test (id integer, a time, b time)
-
-statement ok
-insert into current_time_tzset_test (id, a) values (1, current_time)
+# TODO(otan): this is flaky. Failure can be reproduced with:
```

Comment:
please list the issue number

Reply:
I'd rather not interrupt this build now.
Anyway I've referred to this skip as a comment in #43036.

Reply:
I will do better next time.